models/voicecraft.py

```diff
@@ -1416,7 +1416,13 @@ def sample_helper(n_eog, logits, codebook_eog, top_k, top_p, temperature, prev_t
         return res, flatten_gen[0].unsqueeze(0)
     
 
-class VoiceCraftHF(VoiceCraft, PyTorchModelHubMixin, repo_url="https://github.com/jasonppy/VoiceCraft", tags=["Text-to-Speech", "VoiceCraft"]):
+class VoiceCraftHF(
+        VoiceCraft,
+        PyTorchModelHubMixin,
+        repo_url="https://github.com/jasonppy/VoiceCraft",
+        tags=["Text-to-Speech"],
```

Comment:
```suggestion
        tags=["text-to-speech"],
```
i would also lowercase the tag (and set it as `pipeline_tag` no?)

Reply:
`pipeline_tag` cannot be set like this at the moment to defaulting to `tags`. I opened a quick PR for the future: https://github.com/huggingface/huggingface_hub/pull/2228.